internal/validator/handler.go

```diff
@@ -49,6 +50,8 @@ type enrichedMessageContext struct {
 
 func (this *handler) onMessage(ctx context.Context, msg *kafka.Message) {
 	request := messageModel.IngressValidationRequest{}
+	requestType, _ := kafkaUtils.GetHeader(msg, payloadTypeHeader)
```

Comment:
Since you find out what the request type is here, it would be useful to add that info as a field to the logger stored in context. That way, we'll know what type of upload we're dealing with throughout the logs produced for this payload.